WebDriverAgentTests/IntegrationTests/FBElementSwipingTests.m

```diff
@@ -17,19 +17,38 @@
 
 @interface FBElementSwipingTests : FBIntegrationTestCase
 @property (nonatomic, strong) XCUIElement *scrollView;
+- (void)openScrollView;
 @end
 
 @implementation FBElementSwipingTests
 
+- (void)openScrollView
+{
+  [self launchApplication];
+  [self goToScrollPageWithCells:YES];
+  self.scrollView = [[self.testedApplication.query descendantsMatchingType:XCUIElementTypeAny] matchingIdentifier:@"scrollView"].element;
+}
+
 - (void)setUp
 {
   [super setUp];
-  static dispatch_once_t onceToken;
-  dispatch_once(&onceToken, ^{
-    [self launchApplication];
-    [self goToScrollPageWithCells:YES];
-    self.scrollView = [[self.testedApplication.query descendantsMatchingType:XCUIElementTypeAny] matchingIdentifier:@"scrollView"].element;
-  });
+  if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"15.0")) {
+    [self openScrollView];
```

Comment:
mm, it seems ios 15 simulators could not find an element after actions...
Once change the view, the element could be detected...
I assume this is simulators side issue.

Reply:
After investigation more, I found this issue did not occur when I changed to:
```
- (void)testSwipeUp
{
  [self.testedApplication fb_swipeWithDirection:@"up" velocity:nil];
  FBAssertInvisibleCell(@"0");
}

- (void)testSwipeDown
{
  [self.testedApplication fb_swipeWithDirection:@"up" velocity:nil];
  FBAssertInvisibleCell(@"0");
  [self.testedApplication fb_swipeWithDirection:@"down" velocity:nil];
  FBAssertVisibleCell(@"0");
}

- (void)testSwipeDownWithVelocity
{
  [self.testedApplication fb_swipeWithDirection:@"up" velocity:nil];
  FBAssertInvisibleCell(@"0");
  [self.testedApplication fb_swipeWithDirection:@"down" velocity:nil];
  FBAssertVisibleCell(@"0");
}
```

`mobile:swipe` action also did not have this issue.
So, I assume definitely XCTest's UI interaction stuff